etc/behat/suites.yml

```diff
@@ -34,6 +34,7 @@ imports:
     - suites/ui_cart.yml
     - suites/ui_channel.yml
     - suites/ui_checkout.yml
+    - suites/ui_localization.yml
```

Comment:
I think more appropriate suite name would be ``addressing``, where we will put:

* country management
* zones

Any better ideas? :)